### DISCUSSION TITLE
疫情期間當阿宅，17 歲少年做出意念控制的機械手臂

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#650] [科技新知]
人們COVID-19疫情期間被迫待在家防疫，住在美國維吉尼亞州（Virginia）的17歲高中生崔（Benjamin Choi，音譯）也關在家裡，但可沒一刻閒著。疫情期間，他憑自己的力量打造可用意念控制的機械手臂，成品完成性高且價格實惠，接受《史密森尼雜誌》（Smithsonian magazine）採訪和各家媒體報導。
有這樣的技術及熱情，真令我敬佩，我17歲還不知道在幹嘛XD
https://technews.tw/2022/06/04/17-year-old-kid-genius-built-a-mind-controlled-prosthetic-arm-in-his-spare-time/

**Comment by Jay-722** [comment]
我只能說，同樣17的時候，沒必較沒傷害

**Comment by minniewei** [comment]
嗯嗯嗯....人比人氣死人...所以說我果然是平凡人呀~~~

**Comment by sys511613** [comment]
17歲的時候我連程式語言都還沒接觸，人家就已經做出這種東西，只能說真的是很厲害

**Comment by leon10456** [comment]
感謝樓主分享 我感覺自己越來越費了..

**Comment by Lai27330** [comment]
沒想到藍芽連腦波數據都可以接收，而且透過演算法模擬人體神經訊號來隔空操縱真的太酷了!

**Comment by HaoFish123** [comment]
感謝分享，感覺可以應用在義肢領域上。

**Comment by Jimmy911218** [comment]
這文章最印象深刻的是麻省理工學院願意出錢出力這件事
她告訴我只要在麼些地方乳利可能就會吸引別人來協助你
不管事情大小

**Comment by treeann** [comment]
感謝樓主分享 17歲的差距阿哈哈

**Comment by 110409538** [comment]
可以造福很多人的發明

**Comment by Hchejxnc** [comment]
現在網路資訊發達，將自己努力的結晶上傳到網路，被別人發現的機會就非常高~

**Comment by Evan0979** [comment]
做成武器感覺會很可怕,生化人不再只出現在電影中了

**Comment by Yunhans** [comment]
機器手天使的未來近在眼前

**Comment by danwan1027** [comment]
等我練會念能力也可以做到一樣的事

**Comment by StevenWanglolz** [comment]
真的好強啊~自學python和C++、會寫機器人的驅動程式，而且利用蒐集到的腦波圖來透過AI建模，自己打造出了意念控制的機械手臂!!! 我可能只符合疫情在家當阿宅的部分，沒有辦法做出機械手臂:(

**Comment by williamyen0623** [comment]
太無情了 17歲就有辦法在stackoverflow上面自學 然後還做出這麼革命性的作品 我17歲還在媽媽十塊

**Comment by Doreen331** [comment]
我17歲連hello world都print不出來，疫情關在家把netflix片單快看完＝＝
但這個發明的概念/技術感覺可以賣給一些大公司再作進一步的研發，例如用意識控制電腦，這樣以後遠距上課不用起床就可以進到會議室，然後他17歲就可以財富自由了，我還要當一輩子的社畜😍

**Comment by Kotori0501** [comment]
謝謝分享
我17歲時也很宅，沒接觸過程式語言，更別提自己從零到有製作機器手臂，所以在這個年齡成能有自己的想法，我非常崇敬並以他為效仿對象，順便一提現在疫情期間我依然很宅，也花費的兩年時間什麼作品都沒有產生。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
真的好厲害 17歲的我只會打球打哈哈 對我來說開心才是最重要的 沒想到新聞主角能做出如此成就

**Comment by CPCpyt** [comment]
我想這個技術對於行動不方便的人能夠有很大的幫助，
此外或許能夠對於探討人類腦部活動有更進一步的助力。

**Comment by BlahFish1006** [comment]
真的好佩服...17歲就會自己找資源自學，還可以想到能用EEG來取代侵入式微型感測器，
感覺光是要能看懂數據在表達什麼就是一門大學問了。

**Comment by harrylee1971** [comment]
或許真的有一天可以玩到完全潛行的VR遊戲，這樣子這輩子就值了阿

**Comment by ghost** [comment]
我認為他利用在宅中的時間中運用創意與技術，實現了這一壯舉的這件事，或許代表著能成功的人，往往是善於利用時間的人，而我們之所以會沒沒無聞，不是因為我們愚鈍，只是我們都把時間花在娛樂和讚賞天才

**Comment by wanderer657** [comment]
崔同學在疫情期間就厲害了，他自己做了個能用意念控制的機械手臂！這不只是他個人的成功，也展現了青少年在科技上的超強潛力。這件事也告訴我們，就算在家，高中生也能靠自學和努力做出驚人的技術產品。他的成就不僅激勵了其他青少年，也讓我們看到了科技的無限可能。这也讓人對未來充滿了無限的想像。

**Comment by William08290829** [comment]
對於年紀輕輕就能完成令人驚歎的成就，我感到非常驚訝和欽佩。這表明了人的潛力是無限的，無論年齡大小，只要有足夠的才華、努力和機遇，都能創造出令人矚目的成就。這也提醒著我們，不要輕視任何一個人的能力和潛力，每個人都有可能成為改變世界的力量。同時，對於自己，這也是一種激勵和鞭策，讓我更加努力地追求自己的夢想和目標，不斷挑戰自己的極限，不斷成長和進步。
